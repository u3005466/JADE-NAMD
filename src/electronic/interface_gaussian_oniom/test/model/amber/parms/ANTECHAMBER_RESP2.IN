Resp charges for organic molecule

 &cntrl

 nmol = 1,
 ihfree = 1,
 ioutopt = 1,
 iqopt = 2,
 qwt =  0.00100,

 &end
    1.0
Resp charges for organic molecule
    1    6
    6    0
    1    0
    1    2
    7  -99
    1  -99
    1  -99


