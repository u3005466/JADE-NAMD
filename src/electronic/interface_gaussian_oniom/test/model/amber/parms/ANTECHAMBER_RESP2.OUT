
 -----------------------------------------------
      Restrained ESP Fit 2.3  Amber 4.1
 -----------------------------------------------
 Resp charges for organic molecule                                               
 -----------------------------------------------


 inopt       =     0   ioutopt     =     1
 nmol        =     1   iqopt       =     2
 ihfree      =     1   irstrnt     =     1
 iunits      =     0   qwt         =   0.00100000
 wtmol(1) =     1.000000
 subtitle:

Resp charges for organic molecule                                               
ich =   1  iuniq =   6
    1    6    0
    2    1    0
    3    1    2
    4    7  -99
    5    1  -99
    6    1  -99
 new q0 values to be read   6

  ----------------------------------------------------------------------------
     ATOM                       COORDINATES                         CHARGE
                       X             Y                Z
  ----------------------------------------------------------------------------
  ----------------------------------------------------------------------------


 Charge on the molecule(ich) =    1
 Total number of atoms (iuniq)     =    6
 Weight factor on initial charge restraints(qwt)=     0.10000D-02


  there are  1 charge constraints:


     1       1
     2       1
     3       1
     4       1
     5       1
     6       1

 Reading esp"s for molecule   1
 total number of atoms      =     6
 total number of esp points =  2706


 center     X       Y       Z 
    1   0.0000000E+00   0.0000000E+00  -0.1275351E+01
    2   0.0000000E+00   0.1784805E+01  -0.2293494E+01
    3   0.0000000E+00  -0.1784805E+01  -0.2293494E+01
    4   0.0000000E+00   0.0000000E+00   0.1133148E+01
    5   0.0000000E+00   0.1639269E+01   0.2153532E+01
    6   0.0000000E+00  -0.1639269E+01   0.2153532E+01
 Initial ssvpot =    82.048


 Number of unique UNfrozen centers=    2

 Non-linear optimization requested.
 qchnge =    0.1517499755    
 qchnge =    0.1875402347E-03
 qchnge =    0.5673521424E-06

 Convergence in    2 iterations


Resp charges for organic molecule                                               

          Point Charges Before & After Optimization

    no.  At.no.    q(init)       q(opt)     ivary    d(rstr)/dq
    1   6       0.265605       0.265141      0       0.003529
    2   1       0.199377       0.199609      0       0.000000
    3   1       0.199377       0.199609      2       0.000000
    4   7      -0.542005      -0.542005    -99       0.001814
    5   1       0.438823       0.438823    -99       0.000000
    6   1       0.438823       0.438823    -99       0.000000

 Sum over the calculated charges:      1.000

        Statistics of the fitting:
  The initial sum of squares (ssvpot)                     82.048
  The residual sum of squares (chipot)                     0.004
  The std err of estimate (sqrt(chipot/N))               0.00121
  ESP relative RMS (SQRT(chipot/ssvpot))                 0.00696

 Center of Mass (Angst.):

 X  =   0.00000      Y  =   0.00000      Z  =   0.00000

 Dipole (Debye):

 X  =   0.00000      Y  =   0.00000      Z  =   0.05621

 Dipole Moment (Debye)=   0.05621

 Quadrupole (Debye*Angst.):

 Qxx = -12.82585     QYY =   1.82222     QZZ =  11.00363
 Qxy = -12.82585     QXZ = -12.82585     QYZ = -12.82585
